community/kernel/src/main/java/org/neo4j/graphdb/factory/GraphDatabaseSettings.java

```diff
@@ -466,9 +466,10 @@ private static String defaultPageCacheMemory()
     @Description( "Relationship count threshold for considering a node to be dense" )
     public static final Setting<Integer> dense_node_threshold = setting( "dbms.relationship_grouping_threshold", INTEGER, "50", min(1) );
 
-    @Description( "Log executed queries that takes longer than the configured threshold. "
-            + "_NOTE: This feature is only available in the Neo4j Enterprise Edition_." )
-    public static final Setting<Boolean> log_queries = setting("dbms.logs.query.enabled", BOOLEAN, FALSE );
+    @Description( "Log executed queries that takes longer than the configured threshold, dbms.logs.query.threshold. " +
+            "Log entries are written to the file query.log located in the log directory. " +
```

Comment:
"log directory"

Just checking: is this intended to be a description of the directory (in which case it's fine) or the name of the directory (in which case it should be "logs"). For consistency, would you refer to the directory holding configuration as "the configuration directory" or "the conf directory"?